lua/astrocommunity/pack/typst/init.lua

```diff
@@ -13,4 +13,10 @@ return {
       opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "typst-lsp" })
     end,
   },
+  {
+    "chomosuke/typst-preview.nvim",
+    cmd = { "TypstPreview", "TypstPreviewToggle" },
+    version = "0.1.*",
```

Comment:
This should have an empty opts table

Reply:
Should we perhaps lazy load this on typst files?

Reply:
that does make sense but at the end of they if they want to enable it they have to run the command so i don't think that would make that much difference maybe the start up time would be effected since it has to load it right a way in this case?

Reply:
What is the command for typst-preview.nvim?

Reply:
I did change it to lazy load on filetype but I do believe it doesn't make that much difference since for enabling the preview you have to use the command anyway

Reply:
> What is the command for typst-preview.nvim?

It has quite a bunch but the ones that are capable of starting the functionality are: TypstPreview and TypstPreviewToggle

Reply:
it's now back to lazy load on command